composer.json

```diff
@@ -8,11 +8,11 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1",
         "sonata-project/admin-bundle": "^3",
-        "symfony/yaml": "^3.4",
-        "symfony/finder": "^3.4",
-        "doctrine/doctrine-bundle": "^1.6",
+        "symfony/yaml": "^4.4",
+        "symfony/finder": "^4.4",
+        "doctrine/doctrine-bundle": "^2.0",
```

Comment:
Veel van onze bundles requiren doctrine bundle v1, dus dit levert nu conflicten op. Is er een reden dat je hebt gekozen voor 2?

``shell
$ composer require --no-scripts --no-plugins --no-progress --no-suggest --profile "zicht/messages-bundle:dev-feature/sf4 as 5.0.0"
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for zicht/messages-bundle dev-feature/sf4 as 5.0.0 -> satisfiable by zicht/messages-bundle[dev-feature/sf4].
    - Conclusion: remove doctrine/doctrine-bundle 1.12.0
    - Conclusion: don't install doctrine/doctrine-bundle 1.12.0
    - zicht/messages-bundle dev-feature/sf4 requires doctrine/doctrine-bundle ^2.0 -> satisfiable by doctrine/doctrine-bundle[2.0.0, 2.0.0-Beta1, 2.0.1, 2.0.2, 2.0.3, 2.0.4, 2.0.5, 2.0.6, 2.0.7, 2.0.x-dev].
    - Can only install one of: doctrine/doctrine-bundle[2.0.0, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.0-Beta1, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.1, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.2, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.3, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.4, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.5, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.6, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.7, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.x-dev, 1.12.0].
    - Can only install one of: doctrine/doctrine-bundle[2.0.x-dev, 1.12.0].
    - Installation request for doctrine/doctrine-bundle (locked at 1.12.0) -> satisfiable by doctrine/doctrine-bundle[1.12.0].
```

Reply:
Ik heb een commit toegevoegd die dit weer terugzet naar ^1.6